### PR TITLE
feat: did document update

### PR DESCRIPTION
**Pull request by thackerronak**
## Description
This is an implementation of enhancements: did document fields and "add service entry for token endpoint".

More details in #292, #296

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: miw/src/main/java/org/eclipse/tractusx/managedidentitywallets/constant/StringPool.java [hunk 1]
```diff
@@ -94,4 +94,12 @@ private StringPool() {
     public static final String AS_JWT = "asJwt";
 
     public static final String BPN_CREDENTIAL = "BpnCredential";
+
+    public static final String ASSERTION_METHOD = "assertionMethod";
+    public static final String CONTEXT = "@context";
```
Comment (nitin-vavdiya) [inline miw/src/main/java/org/eclipse/tractusx/managedidentitywallets/constant/StringPool.java:99]: please use constant from JsonLdObject.CONTEXT

File: miw/src/main/java/org/eclipse/tractusx/managedidentitywallets/service/JwtPresentationES256KService.java [hunk 2]
```diff
@@ -130,7 +132,22 @@ public DidDocument buildDidDocument(String bpn, Did did, List<VerificationMethod
...
+        //add assertionMethod
+        List<URI> ids = new ArrayList<>();
+        jwkVerificationMethods.forEach((verificationMethod) -> {
+            ids.add(verificationMethod.getId());
+        });
+        didDocument.put(StringPool.ASSERTION_METHOD, ids);
+        //add service
+        Map<String, Object> serviceData = Map.of(Verifiable.ID, did.toUri()+"#"+StringPool.SECURITY_TOKEN_SERVICE, Verifiable.TYPE, StringPool.SECURITY_TOKEN_SERVICE,
+                StringPool.SERVICE_ENDPOINT,  StringPool.HTTPS_SCHEME + miwSettings.host() + "/api/token");
+        org.eclipse.tractusx.ssi.lib.model.did.Service service1 = new org.eclipse.tractusx.ssi.lib.model.did.Service(serviceData);
```
Comment (nitin-vavdiya) [inline miw/src/main/java/org/eclipse/tractusx/managedidentitywallets/service/JwtPresentationES256KService.java:145]: rename service1 to more meaningful ie. tokenService 
Same for serviceData and serviceData1

**Review by nitin-vavdiya — COMMENTED** [review]
Code changes are fine

It is good if can add support of `service`  and `assertionMthod` inside `DidDocumentBuilder` and `DidDocument` in the SSI lib project.

@borisrizov-zf  Please suggest whether we should merge this PR and letter add support in SSI lib or we should first do changes in SSI lib and publish new version?

File: miw/src/main/java/org/eclipse/tractusx/managedidentitywallets/constant/StringPool.java (hunk 1 above)
Comment (thackerronak) [reply miw/src/main/java/org/eclipse/tractusx/managedidentitywallets/constant/StringPool.java]: Updated!

File: miw/src/main/java/org/eclipse/tractusx/managedidentitywallets/service/JwtPresentationES256KService.java (hunk 2 above)
Comment (thackerronak) [reply miw/src/main/java/org/eclipse/tractusx/managedidentitywallets/service/JwtPresentationES256KService.java]: Updated!

**Review by borisrizov-zf — APPROVED** [review]

**Pull request merged**